<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientVectorBase</class>
 <widget class="ScrPaletteBase" name="GradientVectorBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>163</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gradient Vector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageLinear">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Start:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ScrSpinBox" name="gX1">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="ScrSpinBox" name="gY1">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>End:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ScrSpinBox" name="gX2">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="ScrSpinBox" name="gY2">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Skew:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ScrSpinBox" name="gSk">
         <property name="minimum">
          <double>-89.000000000000000</double>
         </property>
         <property name="maximum">
          <double>89.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRadial">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Start:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ScrSpinBox" name="gX1_2">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="ScrSpinBox" name="gY1_2">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>End:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ScrSpinBox" name="gX2_2">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="ScrSpinBox" name="gY2_2">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Focal:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ScrSpinBox" name="gFX">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="ScrSpinBox" name="gFY">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Skew:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ScrSpinBox" name="gSk_2">
         <property name="minimum">
          <double>-89.000000000000000</double>
         </property>
         <property name="maximum">
          <double>89.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Scale:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="ScrSpinBox" name="gSc">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFourColor">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>1. Point:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ScrSpinBox" name="gC1X">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="ScrSpinBox" name="gC1Y">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>2. Point:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ScrSpinBox" name="gC2X">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="ScrSpinBox" name="gC2Y">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>3. Point:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ScrSpinBox" name="gC3X">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="ScrSpinBox" name="gC3Y">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>4. Point:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ScrSpinBox" name="gC4X">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="ScrSpinBox" name="gC4Y">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDiamond">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>1. Point:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ScrSpinBox" name="gC1XD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="ScrSpinBox" name="gC1YD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>2. Point:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ScrSpinBox" name="gC2XD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="ScrSpinBox" name="gC2YD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>3. Point:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ScrSpinBox" name="gC3XD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="ScrSpinBox" name="gC3YD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>4. Point:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ScrSpinBox" name="gC4XD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="ScrSpinBox" name="gC4YD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>5. Point:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="ScrSpinBox" name="gC5XD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="ScrSpinBox" name="gC5YD">
         <property name="minimum">
          <double>-3000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMesh">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QToolButton" name="editPoints">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Edit Grid Points</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="editControlPoints">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Edit Control Points</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="resetControlPoint">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Reset selected Control Point</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="resetAllControlPoints">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Reset all Control Points</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPushButton" name="buttonNewGrid">
             <property name="text">
              <string>New Mesh...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="buttonResetGrid">
             <property name="text">
              <string>Reset Mesh</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonPoLIne">
             <property name="text">
              <string>Mesh -&gt; Shape</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>62</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ScrPaletteBase</class>
   <extends>QWidget</extends>
   <header>ui/scrpalettebase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">ui/scrpalettebase.h</include>
  <include location="local">ui/scrpalettebase.h</include>
  <include location="local">ui/scrspinbox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
